format.sh

```diff
@@ -132,7 +132,7 @@ elif [[ "$1" == '--all' ]]; then
     pylint "${PYLINT_FLAGS[@]}" sky
 else
     # Pylint only files in sky/ that have changed in last commit.
-    changed_files=$(git diff --name-only --diff-filter=ACM "$MERGEBASE" -- 'sky/*.py' 'sky/*.pyi')
+    changed_files=$(git diff --name-only --diff-filter=ACM "$MERGEBASE" -- 'sky/*.py' 'sky/*.pyi' ':!sky/skylet/**')
```

Comment:
what is this for

Reply:
found some files in `skylet` are not properly formatted.

Reply:
Pls double check.